Lokad.AzureEventStore/Cache/AzureCacheProvider.cs

```diff
@@ -36,7 +36,8 @@ public AzureCacheProvider(CloudBlobContainer container, bool useStateCache, int
 
         private async Task<CloudBlockBlob[]> Blobs(string fullname)
         {
-            await _container.CreateIfNotExistsAsync();
```

Comment:
It seems redundant to call `ExistsAsync()` before calling `CreateIfNotExistsAsync()`. You should add a comment explaining why this call is in fact necessary.

Reply:
Indeed.
I am adding comments and switching the second call to a more explicit CreateAsync() call.
Thanks.